alerts-ui/src/app/home/home.component.html

```diff
@@ -12,8 +12,9 @@ <h2>{{'home.descripion' | translate}}</h2>
       <a href="https://bittrex.com/">BITTREX</a>,
       <a href="https://www.binance.com/">BINANCE</a>,
       <a href="https://bitso.com/">BITSO</a>,
+	  <a href="https://coinmarketcap.com/">COINMARKETCAP</a>,
       <a href="https://hitbtc.com/">HITBTC</a>,
-      <a href="https://www.kucoin.com/">KUCOIN</a>
+      <a href="https://www.kucoin.com/">KUCOIN</a>	  
```

Comment:
Why is this changed?

Reply:
I first added a new link to the end, and then sorted them alphabetically

Reply:
Please, remove the extra spaces to keep the only required line.

Reply:
sure, I already fixed this locally. Now trying to split the branches in order to avoid mixed commits:)

Reply:
An approach is to use `git rebase`, there you could mark the extra commits to be deleted and squash the commits related to this PR, then, do `git push -f`.